pkg/assert/node.go

```diff
@@ -74,18 +74,44 @@ func NodeAssertReadyStatus() NodeAssertFunc {
 
 // CheckComponentCmdNode runs a command on a node and asserts that the value received
 // contains the specified substring.
-func CheckComponentCmdNode(cmd, assert, ip string) error {
-	Eventually(func(g Gomega) {
+func CheckComponentCmdNode(cmd, ip string, asserts ...string) error{
+var err error
```

Comment:
you are doing nothing with this var err

you are creating it outside eventually and returning it directly